### PR TITLE
Implementation to Master PR

### PR DESCRIPTION
**Pull request by AkshaySapra**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## Introduction
+The Linux Cluster Administration (LCA) team manages a Linux cluster of 10 nodes/servers which are running CentOS 7. These servers are internally connected through a switch and able to communicate through internal IPv4 addresses.
+
+The LCA team needs to record the hardware specifications of each node and monitor node resource usages (e.g. CPU/Memory) in realtime. Cluster Monitor Agent is an internal tool that monitors the cluster resources. The collected data is stored in an RDBMS database. LCA team will use the data to generate some reports for future resource planning purposes (e.g. add/remove servers).
+
+
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md:9]: diagram

File: linux_sql/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Usage
+
+1) Run "./scripts/psql_docker.sh start" once to start the database with docker. Then run the ddl.sql file once to initialize the database tables.
+2) `host_info.sh` collects the host hardware info and inserts it into the database. It will be run only once at the installation time.
+3) `host_usage.sh` collects the current host usage (CPU and Memory) and then inserts into the database. It will be triggered by the crontab job every minute.
+4) crontab is setup to run once every minute
+
+## Improvements 
+
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md:44]: 3 future features

File: linux_sql/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+memory_free: Amount of idle memory, measured in MB
+cpu_idle: Percentage of total CPU time spent running kernel/system code
+cpu_kernel: Percentage of total CPU time spent idle
+disk_io: Number of current disk I/O operations in progress
+disk_available: Amount of disk space available MB
+
+
+## Usage
+
+1) Run "./scripts/psql_docker.sh start" once to start the database with docker. Then run the ddl.sql file once to initialize the database tables.
```
Comment (Jarvis-deprecated-account) [inline linux_sql/README.md:38]: format

File: linux_sql/scripts/psql_docker.sh [hunk 4]
```diff
@@ -0,0 +1,76 @@
+su centos
+
+#psql docker docs https://hub.docker.com/_/postgres
+export PGPASSWORD='password'
+#run psql
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/psql_docker.sh]: cleanup comments

File: linux_sql/scripts/psql_docker.sh [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+		exit 0;
+	fi
+	# if the volume has not been created
+	if [[ "$(docker volume ls | egrep "pgdata1$" | wc -l)" == 0 ]];
+	then
+		# delete the container named jrvs-psql if there is one
+		if [[ "$(docker ps -a -f name=jrvs-psql | wc -l)" == 2 ]];
+		then
+			docker stop jrvs-psql
+			docker container rm jrvs-psql
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/psql_docker.sh]: don't remove

File: linux_sql/scripts/psql_docker.sh [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+	else
+		docker start jrvs-psql
+		exit 0
+	fi
+# check if the command is to stop 
+elif [[ "$1" == "stop" && "$#" == 1 ]];
+then
+	#stop the container & the docker
+	systemctl status docker && docker stop jrvs-psql
+	systemctl status docker && systemctl stop docker
```
Comment (Jarvis-deprecated-account) [inline linux_sql/scripts/psql_docker.sh]: don't stop docker sever

**Review by ghost — CHANGES_REQUESTED** [review]

File: linux_sql/README.md (hunk 1 above)
Comment (AkshaySapra) [reply linux_sql/README.md:9]: done

**Pull request merged**